### PR TITLE
total nodes number displaying for clusters was implemented

### PR DESCRIPTION
**Pull request by tengu-alt**

### CODE REVIEW

File: apis/clusters/v1beta1/cassandra_types.go [hunk 1]
```diff
@@ -112,6 +113,19 @@ func (s *CassandraStatus) DataCentresEqual(o *CassandraStatus) bool {
...
+	for _, dc := range cs.DataCentres {
+		for _, node := range dc.Nodes {
+			total++
+			if node.Status == models.RunningStatus {
+				running++
+			}
+		}
+	}
+	cs.NodeCount = fmt.Sprintf("%v/%v", running, total)
+}
```
Comment (ribaraka) [inline apis/clusters/v1beta1/cassandra_types.go:127]: you use the same body func for all cluster CRDs. Please make this func generic

**Review by ribaraka — CHANGES_REQUESTED** [review]

**Review by ribaraka — APPROVED** [review]

**Review by worryg0d — APPROVED** [review]

File: pkg/instaclustr/mock/server/go/model_addon_bundles_options.go [hunk 2]
```diff
@@ -131,10 +131,10 @@ type AddonBundlesOptions struct {
 	// Accepts true/false. Enables Password Authentication and User Authorization.
 	PasswordAuth bool `json:"passwordAuth,omitempty"`
 
-	// The number of Redis Master Nodes to be created.
+	// The number of Redis Master NodeCount to be created.
 	MasterNodes int32 `json:"masterNodes,omitempty"`
 
-	// The number of Redis Replica Nodes to be created.
+	// The number of Redis Replica NodeCount to be created.
```
Comment (testisnullus) [inline pkg/instaclustr/mock/server/go/model_addon_bundles_options.go:137]: It's not critical, but let's revert these changes

File: apis/clusters/v1beta1/generic_status.go [hunk 3]
```diff
@@ -90,3 +91,16 @@ func (n *Node) FromInstAPI(instaModel *models.Node) {
...
+//	for _, dc := range status.DataCentres {
+//		for _, node := range dc.Nodes {
+//			total++
+//			if node.Status == models.RunningStatus {
+//				running++
+//			}
+//		}
+//	}
+//	ks.NodeCount = fmt.Sprintf("%v/%v", running, total)
+//}
```
Comment (testisnullus) [inline apis/clusters/v1beta1/generic_status.go:106]: Let's remove it if unused

**Review by testisnullus — CHANGES_REQUESTED** [review]

File: apis/clusters/v1beta1/generic_status.go (hunk 3 above)
Comment (tengu-alt) [reply apis/clusters/v1beta1/generic_status.go]: agree, thank you

**Review by testisnullus — APPROVED** [review]

**Pull request merged**